util/util_test.go

```diff
@@ -80,3 +83,59 @@ func TestSecretCopy(t *testing.T) {
 		})
 	}
 }
+
+// TestGenerateCacheKey tests the GenerateCacheKey function
+func TestGenerateCacheKey(t *testing.T) {
+	// Define test cases
+	testCases := []struct {
+		format    string
+		args      []any
+		expected  string
+		shouldErr bool
+	}{
+		{
+			format:    "Hello %s",
+			args:      []any{"World"},
+			expected:  generateExpectedKey("Hello World"),
```

Comment:
Instead of creating a new method that has the duplicated business logic in the test file, use the expected string value directly.

Reply:
Done